src/Core/Components/AnchoredRegion/FluentAnchoredRegion.razor.cs

```diff
@@ -140,4 +149,49 @@ public partial class FluentAnchoredRegion : FluentComponentBase
     /// </summary>
     [Parameter]
     public RenderFragment? ChildContent { get; set; }
+
+    /// <summary>
+    /// Gets or sets whether the element should receive the focus when the component is loaded.
+    /// </summary>
+    [Parameter]
+    public bool AutoFocus { get; set; } = false;
+
+    /// <summary />
+    protected async override Task OnAfterRenderAsync(bool firstRender)
+    {
+        if (firstRender)
+        {
+            Module ??= await JSRuntime.InvokeAsync<IJSObjectReference>("import", JAVASCRIPT_FILE);
+
+            // Go to the first Focusable element (after a delay of 100ms, to allow the DOM to render the Web Components)
+            if (AutoFocus)
+            {
+                await Module.InvokeVoidAsync("goToNextFocusableElement", Element, false, 100);
```

Comment:
100ms seems a bit long to me. Can we make it shorter?

Reply:
We can, but I'm afraid the Web Component won't be created. 
And it's only used on the first call (after, the time is not set), when the user performs a manual action, so it's not perceptible to the eye.